wave_vision/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.3)
 PROJECT(wave_vision)
+# SET(CMAKE_BUILD_TYPE RELEASE)
```

Comment:
Oops. I had that for testing the speed of some algorithms during testing. Thanks!